scripts/script.js

```diff
@@ -13,14 +13,18 @@ const pieces = new SharedArray('pieces', function () {
   return arr // f must be an array[]
 })
 
-const timeBoost = new Trend('time-boost')
-const ttfbBoost = new Trend('ttfb-boost')
-const timeRaw = new Trend('time-raw')
-const ttfbRaw = new Trend('ttfb-raw')
-const timeDelta = new Trend('time-delta')
-const ttfbDelta = new Trend('ttfb-delta')
-const boostSuccess = new Rate('success-boost')
-const rawSuccess = new Rate('success-raw')
+const bandwidthBoost = new Trend('bandwidth_boost')
+const bandwidthRaw = new Trend('bandwidth_raw')
+const dataReceivedBoost = new Counter('data_received_boost')
```

Comment:
what's the difference between a `Counter` and a `Trend` when we're only calling `.add()` once?

Reply:
Counters will total to a sum, while trends will calculate stats over time like average and median. The metrics are actually added across concurrent runs.